### PR TITLE
#8 Add all required models

### LINKED ISSUE #8: Add all required models

### PR DESCRIPTION
**Pull request by NMishchenko**

### CODE REVIEW

File: cinome/cinomedjango/cinomeapp/models.py [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
+
+
+class UserFriends(models.Model):
+    user = models.ForeignKey(User, on_delete=models.RESTRICT)
+    friend = models.ForeignKey(User, on_delete=models.RESTRICT, related_name="friend_relation")
+
+
+class ListType(models.Model):
+    name = models.CharField(max_length=255)
+
```
Comment (volodymyr-riabenko) [inline cinome/cinomedjango/cinomeapp/models.py]: I don't think that these models should be in one file

**Review by volodymyr-riabenko — CHANGES_REQUESTED** [review]

File: cinome/cinomedjango/cinomeapp/models.py (hunk 1 above)
Comment (NMishchenko) [reply cinome/cinomedjango/cinomeapp/models.py]: I don't know which version is correct, but in many repositories, including the official documentation, all models are placed in models.py.

File: cinome/cinomedjango/cinomeapp/models.py [hunk 2]
```diff
@@ -1,7 +1,59 @@
...
+    email = models.EmailField()
+    password = models.CharField(max_length=255, validators=[MinLengthValidator(4)])
+    username = models.CharField(max_length=32, validators=[MinLengthValidator(4)], unique=True)
+    image_url = models.FilePathField(blank=True)
+    is_email_confirmed = models.BooleanField(default=False)
+
+
+class UserFriends(models.Model):
+    user = models.ForeignKey(User, on_delete=models.RESTRICT)
+    friend = models.ForeignKey(User, on_delete=models.RESTRICT, related_name="friend_relation")
```
Comment (AlexandrTsV) [inline cinome/cinomedjango/cinomeapp/models.py]: Do we really need to create such models without any additional logic? I see a lot of navigation properties throughout the whole file. Maybe we could somehow specify this table should also be created? I believe there should be a similar approach to the one we used in EF.

**Review by volodymyr-riabenko — APPROVED** [review]

File: cinome/cinomedjango/cinomeapp/models.py (hunk 1 above)
Comment (volodymyr-riabenko) [reply cinome/cinomedjango/cinomeapp/models.py]: OK! 😎

File: cinome/cinomedjango/cinomeapp/models.py (hunk 2 above)
Comment (NMishchenko) [reply cinome/cinomedjango/cinomeapp/models.py]: Yes, you're right. I'll fix it.

**Review by volodymyr-riabenko — APPROVED** [review]

**Review by AlexandrTsV — APPROVED** [review]

**Pull request merged**